### ISSUE TITLE
[Feature Request] Restore the separated stop and pause buttons, or add option to stop recording with one tap instead of pause

### ISSUE THREAD

**Issue by lrq3000** [Dimowner/AudioRecorder#129]
Dear Dimowner,

Thank you so much for your amazing app, I am using it since a while now, maybe 1.5 year or so? It has been extremely useful, with an extremely good quality and very robust recording especially for very long sessions, that other apps cannot sustain usually (they get closed or the recording is messed up or gets too big).

I would like to request the restoration of the separate stop and pause buttons, as in previous versions, so that a recording can be stopped with a single tap instead of 2.

The reason for this feature request is that I use this app in 2 main settings: for meetings, and while driving (to record thoughts). While driving, one tap to record and one tap to stop recording is OK, but two consecutive precise taps to stop recording is extremely risky.

Alternatively, an option could be added to stop the recording at the first tap. This would work fine for me too, and would allow to keep the current UX, while users with similar needs as me would still be able to restore the old behavior of stopping a recording with a single tap.

Thank you very much in advance for considering my feature request.

**Comment by Dimowner** [comment]
Hello Stephen, thank you for feedback! That is a really good point. Definitely, we need to allow users to stop recording in 1 click. Will try to figure out it.

**Comment by jdalbey** [comment]
I have a suggestion to solve this issue.  Currently while recording is happening the circular button is labeled "STOP" but when you press it the status becomes "Paused".  The button behaves more like a traditional pause button not a stop button. My suggestion is to have two buttons, one labeled PAUSE that behaves the way the current button does, and a second button labeled "STOP" that actually finishes the recording and thus eliminates the need for a "Finish" button. (I've seen other recorder apps with this style of UI and it works well.)

**Comment by lrq3000** [comment]
@jdalbey That would work for me, as long as both buttons are big enough, so that I can tap imprecisely while driving!

**Comment by Dimowner** [comment]
The new rewritten app V2 allows stopping recording with one click.

<img width="340" alt="Image" src="https://github.com/user-attachments/assets/00159875-c248-44d7-ac41-7803ac0f41cf" />

**Comment by lrq3000** [comment]
Yes I saw it, I confirm it works great, thank you very much for adding it !
